.github/workflows/notebooks-digest-updater-upstream.yaml

```diff
@@ -0,0 +1,109 @@
+# The aim of this GitHub workflow is to update the params.env file with the latest digest.
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        required: true
+        description: "Provide the name of the branch you want to update ex main, vx.xx etc"
```

Comment:
A small change. Could we consider updating the notation from `vx.xx` to `YYYYx` in the description to align it with the repository's structure?